ui/components/app/advanced-gas-controls/advanced-gas-controls.component.js

```diff
@@ -68,7 +68,6 @@ export default function AdvancedGasControls({
             }}
             value={maxPriorityFee}
             detailText={maxPriorityFeeFiat}
-            numeric
```

Comment:
This flag does more than set the `type` of the input element to `number`. It also ensures the `onChange` value is a number, and might do other things as well.

If we're considering removing `type=number`, we should probably remove it from the `NumericInput` component.

Reply:
I agree with @Gudahtt here -- I don't believe we want to remove this.